storage/CMakeLists.txt

```diff
@@ -37,6 +37,7 @@ else()
     add_subdirectory(blockdevice)
```

Comment:
On [Line 36](https://github.com/ARMmbed/mbed-os/blob/d4559b8f3db5df5f26c48cbd0f45011925b0f010/storage/CMakeLists.txt#L36) small addition to documentation to reflect changes:
```
-     # Add these subdirectories for the Unit test
+     # Add these subdirectories for the Unit or Greentea test
```